packages/wrangler/src/dev/remote.tsx

```diff
@@ -68,33 +68,39 @@ interface RemoteProps {
 		| undefined;
 	sourceMapPath: string | undefined;
 	sendMetrics: boolean | undefined;
+
+	setAccountId: (accountId: string) => void;
+	experimentalDevenvRuntime: boolean;
 }
 
 export function Remote(props: RemoteProps) {
-	const [accountId, setAccountId] = useState(props.accountId);
 	const accountChoicesRef = useRef<Promise<ChooseAccountItem[]>>();
 	const [accountChoices, setAccountChoices] = useState<ChooseAccountItem[]>();
 
-	useWorker({
-		name: props.name,
-		bundle: props.bundle,
-		format: props.format,
-		modules: props.bundle ? props.bundle.modules : [],
-		accountId,
-		bindings: props.bindings,
-		assetPaths: props.assetPaths,
-		isWorkersSite: props.isWorkersSite,
-		compatibilityDate: props.compatibilityDate,
-		compatibilityFlags: props.compatibilityFlags,
-		usageModel: props.usageModel,
-		env: props.env,
-		legacyEnv: props.legacyEnv,
-		host: props.host,
-		routes: props.routes,
-		onReady: props.onReady,
-		sendMetrics: props.sendMetrics,
-		port: props.port,
-	});
+	if (!props.experimentalDevenvRuntime) {
```

Comment:
```suggestion
	if (!props.experimentalDevEnvRuntime) {
```

Reply:
isn't this breaking existing behaviour, with us now requiring the `experimentalDevEnvRuntime` flag to be set in order to run `useWorker()`?

Reply:
The flag has to be NOT set to run useWorker